doc.go

```diff
@@ -455,6 +455,41 @@ of the returned value:
 	    }
 	}
 
+## Arrow batches
+
+There is a possibility to retrieve data in a columnar format similar to native server response.
```

Comment:
You can retrieve data in a columnar format similar to the format a native server returns.

Reply:
Done. I didn't like the native in such phrase, so I removed it.